### PR TITLE
Create Get-FileSensitivityLabelInfo

### PR DESCRIPTION
**Pull request by SanjoyanM**
The cmdlet and its output looks like this...

image

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/Get-FileSensitivityLabelInfo
+applicable: SharePoint Online
+title: GetFileSensitivityLabelInfo
+schema: 2.0.0
+author: Sanjoyan
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:8]:
```suggestion
author: Sanjoyanm
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ms.author: samust
+ms.reviewer:
+---
+
+# Get-FileSensitivityLabelInfo
+
+## SYNOPSIS
+
+Extracts and displayes the sensitivity label related information attached to a office file stores in SharePoint
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:17]:
```suggestion
Extracts and displays the sensitivity label related information attached to an office file stored in SharePoint 
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+# Get-FileSensitivityLabelInfo
+
+## SYNOPSIS
+
+Extracts and displayes the sensitivity label related information attached to a office file stores in SharePoint 
+
+## SYNTAX
+
+```powershell
+Get-FileSensitivityLabelInfo -FileUrl <absolute path for file>
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:22]:
```suggestion
Get-FileSensitivityLabelInfo -FileUrl <String> [<CommonParameters>]
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+## DESCRIPTION
+
+The `Get-FileSensitivityLabelInfo` cmdlet runs on a single office online file. If the file has a sensitivity label attached then it returns the id, displayname, isProtectionEnabled flag and id of the parent label(if applicable). You must be a SharePoint Online administrator or Global Administrator to run the `GetFileSensitivityLabelInfo` cmdlet. Note, this cmdlet do not work on files that have labels with custom permission or user defined permission or double key encryption.
+For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps).
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:28]:
```suggestion
For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## INPUTS
+
+## OUTPUTS
+
+## NOTES
+
+## RELATED LINKS
+
+[Getting started with SharePoint Online Management Shell](https://docs.microsoft.com/powershell/sharepoint/sharepoint-online/connect-sharepoint-online?view=sharepoint-ps)
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:63]:
```suggestion
[Getting started with SharePoint Online Management Shell](/powershell/sharepoint/sharepoint-online/connect-sharepoint-online)
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## SYNTAX
+
+```powershell
+Get-FileSensitivityLabelInfo -FileUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Get-FileSensitivityLabelInfo` cmdlet runs on a single office online file. If the file has a sensitivity label attached then it returns the id, displayname, isProtectionEnabled flag and id of the parent label(if applicable). You must be a SharePoint Online administrator or Global Administrator to run the `GetFileSensitivityLabelInfo` cmdlet. Note, this cmdlet do not work on files that have labels with custom permission or user defined permission or double key encryption.
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:27]:
```suggestion
The `Get-FileSensitivityLabelInfo` cmdlet runs on a single office online file. If the file has a sensitivity label attached then it returns the id, displayname, isProtectionEnabled flag and id of the parent label(if applicable). You must be a SharePoint Online administrator or Global Administrator to run the `GetFileSensitivityLabelInfo` cmdlet. Note, this cmdlet does not work on files that have labels with custom permission or user defined permission or double key encryption.
```

**Review by get-itips — APPROVED** [review]
Thanks @SanjoyanM

File: sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+author: Sanjoyanm
+ms.author: samust
+ms.reviewer:
+---
+
+# Get-FileSensitivityLabelInfo
+
+## SYNOPSIS
+
+Extracts and displays the sensitivity label related information attached to an office file stored in SharePoint
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-FileSensitivityLabelInfo:17]:
```suggestion
Extracts and displays the sensitivity label related information attached to an office file stored in SharePoint.
```

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**